docs/content/docs/connectors/table/filesystem.md

```diff
@@ -328,15 +328,30 @@ Time extractors define extracting time from partition values.
         <td>String</td>
         <td>The extractor class for implement PartitionTimeExtractor interface.</td>
     </tr>
+    <tr>
+        <td><h5>partition.time-extractor.formatter-pattern</h5></td>
+        <td style="word-wrap: break-word;">yyyy-MM-dd&nbsp;HH:mm:ss</td>
+        <td>String</td>
+        <td>When 'partition.time-extractor.kind' is set to 'default', you can specify a formatter pattern to get a timestamp from partitions. <br>By default, the format is 'yyyy-MM-dd&nbsp;HH:mm:ss'.
+            <br>Examples as follows:
+            <br>'yyyy-MM-dd' -> '2018-07-14'
+            <br>'dd-MMM-yyyy' -> '14-Jul-2018'
+            <br>'E, MMM dd yyyy' -> 'Sat, Jul 14 2018'
+            <br>More details: https://www.w3.org/QA/Tips/iso-date</td>
```

Comment:
The timestamp-formatter is compatible with Java's [DateTimeFormatter](https://docs.oracle.com/javase/8/docs/api/index.html).